spec/controllers/collaborators_controller_spec.rb

```diff
@@ -88,6 +87,96 @@
         expect(controller).to receive(:remove_collaborator).with(collaborator)
         delete :destroy, id: collaborator, format: :js
       end
+
+      context 'removing a group of collaborators' do
+        let!(:group_member1) { create(:group_member) }
+        let!(:group) { group_member1.group }
+        let!(:group_member2) { create(:group_member, group: group) }
+
+        let!(:collaborator1) { create(:cookbook_collaborator, group: group, user: group_member1.user, resourceable: cookbook) }
+        let!(:collaborator2) { create(:cookbook_collaborator, group: group, user: group_member2.user, resourceable: cookbook) }
+
+        let(:group_resource) { create(:group_resource, resourceable: cookbook, group: group) }
+        let(:group_resources) { GroupResource.where(group: group, resourceable: cookbook) }
+
+        let(:collaborators) { Collaborator.where(group: group, resourceable: cookbook) }
+
+        before do
+          sign_in fanny
+        end
+
+        it 'finds the correct group' do
+          expect(Group).to receive(:find).and_return(group)
```

Comment:
For this to test the correctness of the `find`, I think it needs a `.with(group.id)` included in the expectation. In the current form, the expectation acts more like a stubbed `Group.find` method.